CMakeLists.txt

```diff
@@ -65,10 +65,11 @@ endif()
 #TODO: Contracts' tests depend on different libraries for different build types:
 # libfc.a (for release) or libfc_debug.a (for debug) (the same for secp256k1 library).
 # (see EosioTester.cmake)
+set(TEST_BUILD_TYPE RelWithDebInfo)
+#~set(TEST_BUILD_TYPE ${CMAKE_BUILD_TYPE})
```

Comment:
comment ?

Reply:
Commented version should be left after adding both variants -- debug & releare -- for libfc and libsecp256k1 libraries to the daocasino/daobet-with-cdt docker image.